practice-two/src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -4,13 +4,13 @@ import { Button } from '@components';
 import { BUTTON_VARIANTS } from '@constants/buttons';
 import { editIcon, deleteIcon } from '@assets/images';
 
-const ContextMenu = ({ handleShowCustomerForm }) => {
+const ContextMenu = ({ handleToggleForm }) => {
```

Comment:
I did you tell me before don't use "handle" name for prop

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/a94fb2b9-d89d-4320-8819-7e5b49450246)